packages/@react-spectrum/tree/chromatic/TreeView.chromatic.tsx

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {ActionGroup, Item} from '@react-spectrum/actiongroup';
+import {ActionMenu} from '@react-spectrum/menu';
+import Add from '@spectrum-icons/workflow/Add';
+import {Content, View} from '@react-spectrum/view';
+import Delete from '@spectrum-icons/workflow/Delete';
+import Edit from '@spectrum-icons/workflow/Edit';
+import FileTxt from '@spectrum-icons/workflow/FileTxt';
+import Folder from '@spectrum-icons/workflow/Folder';
+import {generatePowerset} from '@react-spectrum/story-utils';
+import {Grid, repeat} from '@react-spectrum/layout';
+import {Heading, Text} from '@react-spectrum/text';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import {Meta} from '@storybook/react';
+import React from 'react';
+import {SpectrumTreeViewProps, TreeView, TreeViewItem} from '../src';
+
+let states = [
+  {selectionMode: ['multiple', 'single']},
+  {disabledBehavior: ['none', 'all']},
+  {selectionStyle: ['checkbox', 'highlight']}
+];
+
+let combinations = generatePowerset(states);
+let chunkSize = Math.ceil(combinations.length / 3);
+let combo1 = combinations.slice(0, chunkSize);
+let combo2 = combinations.slice(chunkSize, chunkSize * 2);
+let combo3 = combinations.slice(chunkSize * 2, chunkSize * 3);
+
+function shortName(key, value) {
+  let returnVal = '';
+  switch (key) {
+    case 'selectionMode':
+      returnVal = `sm: ${value === undefined ? 'none' : value}`;
+      break;
+    case 'disabledBehavior':
+      returnVal = `disB: ${value}`;
+      break;
+    case 'selectionStyle':
+      returnVal = `secStyle: ${value}`;
+      break;
+  }
+  return returnVal;
+}
+
+const meta: Meta<SpectrumTreeViewProps<object>> = {
+  title: 'TreeView',
+  component: TreeView,
+  parameters: {
+    chromaticProvider: {colorSchemes: ['light'], locales: ['en-US'], scales: ['medium'], disableAnimations: true},
```

Comment:
Ah, I was mimicking TableView's setup since historically adding the colorSchemes/locale/scales/ multiplies the number of concurrently rendered components to the point that the chromatic test becomes really flaky. I've add some more chromatic tests covering those case in a followup